[orgs] Add blogpost: Good, Better, Best. (#14111)
devopsdays/devopsdays-web: content/blog/good-better-best.md

+++
Title = "Good, Better, Best: How to budget for success"

> Date = "2024-05-23T00:00:00-00:00"

Author: ```suggestion
Date = "2024-06-06T00:00:00-00:00"
```

+++
Title = "Good, Better, Best: How to budget for success"
Date = "2024-05-23T00:00:00-00:00"
Author = "DevOpsDays Global Core Organizers"
+++

DevOpsDays events have been running for nearly 15 years now (plug: [come to the
anniversary event in
Antwerp!](https://devopsdays.org/events/2024-antwerp/welcome/)), but the
ecosystem in which events are run has changed. The cost of living has increased
in many countries, the ability to travel has been severely reduced, and there is
less financial ability by many historically generous sponsor companies. 

As one of the co-chairs of the global core team, I wanted to take this
opportunity to highlight a new section we’ve added to the Organizing Guide:
[Good, Better, Best
Budget](https://devopsdays.org/organizing#good-better-best-budget).

Historically, global core organizers have not provided any financial support to

> individual city events. This isn’t changing. However, we would like to at least

Reviewer: May be worth clarifying _why_ we don't provide support. Remember: the public at large doesn't generally understand that we're community/distributed/zero-profit (and, frankly, not all organisers do either).

Author: Would love if you could put some words in here how you would phrase it (I'm not completely sure I understand it either, other than "always been that way")

Reviewer: It's "always been that way" because the global core doesn't really have a reliable source of independent income. 😅 We're effectively a charity-case that Chicago and Amsterdam hand some cash to in order to keep the lights on.

> Historically, global core organizers have not provided any financial support to
> individual city events. This isn’t changing. However, we would like to at least
> help events (especially newer ones who may not have their own processes in place
> for this) consider what would make their event successful, in terms of the
> number of attendees, the facilities offered, and the amount of sponsors that
> takes to happen. 

Reviewer: Maybe just drop the preamble entirely? Also, this document is effectively about financial viability, so we should focus on that terminology (i.e. "success" means different things to different people).

```suggestion
This section is all about helping events—especially newer ones who may not have their own processes in place for this—to understand what would make their event financially viable, in terms of the number of attendees, the facilities offered, and the amount of sponsorship necessary to make these things happen. 
```

coffee is directly invoiced to the sponsor? Also, if your event has been running
for a number of years, think about how many of your sponsorship tiers may change
price. Also, think about how the tiers in your system work, and the cost of your
time to acquire said sponsors: if you land a “gold” sponsor, how many “bronze”
sponsors is that equivalent to? 

In all of this, if things aren’t trending how you think they should be, consider
when you would want to set your go/no-go dates. It’s always a difficult decision
to cancel an event, but if you consider what your lowest “good” level is, you
can drop services from your event before canceling entirely. A major cost is
venue catering: can you drop lunch, and add an extra half hour in your schedule
for attendees to find their own sandwiches? Are you running in a location where
it’s practical for your event to blend into the facilities for office workers
seeking snacks, as not to overload any one particular venue, so everyone has
options and can get back with time to spare before open spaces? 

We would love to hear more about how cities have been running. In that vein,

> we’re piloting a “post event survey”, where we’re asking events to share nuances

Reviewer: Do we know when / who / how / etc for the survey?

about their event and how it ran. We know that most if not all devopsdays events
use the devopsdays.org website, so we can see how many sponsors and speakers
each event attracted, along with ticket pricing, but often the total number of
attendees and how the event actually felt on the ground, including attendees,

> sponsors, and organizers. We hope this information will help us tailor guidance

Reviewer: This is a long sentence and it doesn't appear to terminate properly (?). Consider breaking it into two sentences and terminating the second.

Author: ```suggestion
about their event and how it ran. We know that most if not all devopsdays events
use the devopsdays.org website, so we can see how many sponsors and speakers
each event attracted, along with ticket pricing. But often the total number of
attendees and how the event actually felt on the ground, including attendees,
sponsors, and organizers. We hope this information will help us tailor guidance
```
